src/create-with-theme.js

```diff
@@ -1,31 +1,26 @@
 const React = require('react');
-const PropTypes = require('prop-types');
 const channel = require('./channel');
+const createThemeListener = require('./create-theme-listener');
 
 function createWithTheme(CHANNEL = channel) {
+  const themeListener = createThemeListener(CHANNEL);
   return Component =>
     class withTheme extends React.Component {
```

Comment:
`withTheme` lower case is the function you return on ln 7, which accepts a component, don't mix it up with the class name.